pymc/tests/test_variational_inference.py

```diff
@@ -103,26 +101,13 @@ def three_var_model():
     [
         (not_raises(), {MeanFieldGroup: None}),
         (not_raises(), {FullRankGroup: None, MeanFieldGroup: ["one"]}),
-        (
-            not_raises(),
-            {MeanFieldGroup: ["one"], FullRankGroup: ["two"], NormalizingFlowGroup: ["three"]},
```

Comment:
It would be better to replace normalizing flow group with e.g. `MeanFieldGroup`. The test checks initialization API

Reply:
I tried to replace with `MeanFieldGroup` but `test_init_groups` gave it an error `pymc.variational.opvi.GroupError: No approximation is specified for the rest variables`. Is there any other way to fix this ?